mvg/analysis_classes.py

```diff
@@ -245,8 +325,8 @@ def to_df(self, file_name=None):
         self.check_status()
         if file_name is not None:
             print(f"Saving {self.feature()} data frame results to", file_name)
-            self.dframe.copy().to_csv(file_name, index=False)
-        return self.dframe
+            self._dframe.copy().to_csv(file_name, index=False)
+        return self._dframe
 
     # Save self as pickel
     def to_json(self, file_name=None, raw=False):
```

Comment:
Consider renaming to `save_json` for consistency

Reply:
OK